app/src/main/java/org/oppia/app/testing/AudioFragmentTestActivity.kt

```diff
@@ -12,6 +16,16 @@ class AudioFragmentTestActivity : InjectableAppCompatActivity() {
   override fun onCreate(savedInstanceState: Bundle?) {
     super.onCreate(savedInstanceState)
     activityComponent.inject(this)
-    audioFragmentTestActivityController.handleOnCreate()
+    val internalProfileId =
+      intent.getIntExtra(KEY_AUDIO_FRAGMENT_TEST_PROFILE_ID,/* defaultValue= */ 0)
```

Comment:
Nit change, space after `,`
`intent.getIntExtra(KEY_AUDIO_FRAGMENT_TEST_PROFILE_ID, /* defaultValue= */ 0)
 `

Reply:
The default value should be `-1`

Reply:
Done

Reply:
As mentioned in first comment, there is one spacing nit also. Space after `,`

`intent.getIntExtra(KEY_AUDIO_FRAGMENT_TEST_PROFILE_ID, /* defaultValue= */ 0)`

Reply:
Done.